pkg/chrootarchive/diff_unix.go

```diff
@@ -49,6 +51,10 @@ func applyLayer() {
 		fatal(err)
 	}
 
+	if inUserns {
+		options.InUserNS = true
```

Comment:
That was my fault -- he was testing on essentially ab8e54bbea5248d71c932a4b21696e42292e13bb, which is missing 45afdc9af9c300887cfd9952e04f5ba7d500f4d9 (which fixes the panic).